client/app/components/proposed-development-types-list.hbs

```diff
@@ -0,0 +1 @@
+{{this.proposedDevelopmentTypes}}
```

Comment:
![image](https://user-images.githubusercontent.com/5004319/83068815-e3474b80-a036-11ea-83c3-cd2c067dc501.png)
It's telling me there's no newline — should we follow this rule?

Reply:
yes 👍